docs/intro/ff-ui/toolbar.md

```diff
@@ -43,6 +43,123 @@ The Command Palette makes it easy to find and use things in the App Builder. Ope
 
 ![command-palette.avif](imgs/command-palette.avif)
 
+## AI Tools
+FlutterFlow AI Tools is a powerful set of features designed to speed up app development by leveraging artificial intelligence. These tools allow you to quickly generate app elements such as pages, components, and even entire page layouts with relevant widgets automatically populated.
+
+- **New Page Creation:** Instantly create complete app pages with the necessary layout and widgets. This can be particularly useful when you're unsure where to start.
```

Comment:
can they be part of their own H3? its easier for search, and with arcades, they are big enough as H3

Reply:
though I wonder, at some point, should we have a separate page for our AI suite? since there are so many now.

Reply:
> can they be part of their own H3? its easier for search, and with arcades, they are big enough as H3

Done.



> though I wonder, at some point, should we have a separate page for our AI suite? since there are so many now.

Definitely, In the old docs we had a separate page only.

Reply:
yeah I mean all these new options are in the toolbar and in the old product, the AI stuff was part of it as a part of the features so it made sense to have them be part of separate feature docs. but since the new AI fetaures are more global, they can be added in a separate page and the rest of page gen, scheme gen stuff can be referenced in that page.